_data/agencies.yml

```diff
@@ -42,6 +42,11 @@
   acronym: DOJ
   logo: assets/img/logos/agencies/USDOJ.png
   featured: true
+-
+  name: Department of Labor
+  acronym: DOL
+  logo: assets/img/logos/agencies/USDOL.svg
+  featured: false
```

Comment:
Setting this to `false` explicitly adds clutter without adding value, since it's implicitly false. Please delete L49 and we'll be all set!